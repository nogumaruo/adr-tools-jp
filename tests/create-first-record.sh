adr new The First Decision
cat doc/adr/001-the-first-decision.md
