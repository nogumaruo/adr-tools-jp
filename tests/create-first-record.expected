adr new The First Decision
doc/adr/001-the-first-decision.md
cat doc/adr/001-the-first-decision.md
# 1. The First Decision

日付: 1992-01-12

決定者: @xxx, @xxx

## ステータス

提案中

## コンテキストと問題点の説明

```記載後は削除してください。
この決定を必要とする理由は何ですか？
この決定がない場合の問題点は何ですか？
プロジェクトを取り巻く技術・社会・政治的な背景は何ですか？
```

## 決定要因

```記載後は削除してください。
何が決定の後押しになるのか？を記載
例）
○○を使用した場合コストに比べて効果が上回るか？
○○を使用した場合既存より品質は向上するか？
○○を使用した場合納期は早くなるのか？
etc.
```

## 検討した選択肢

```記載後は削除してください。
何が決定の後押しになるのか？を記載
とり得る選択肢
箇条書きレベルでOK。
例）
○○を使用する
○○を使用しない
```

## 決定事項

```記載後は削除してください。
実現方法
採用する技術・設計思想（言語・ライブラリ・フレームワーク など）
決定に至った理由や背景
採用しなかった技術や設計に対する理由
```

## 検討したこと

```記載後は削除してください。
選択肢を考えるにあたって検討したことを記載（複数記載OK）
コンフルのリンクを記載しても良い
例）
○○ではなくて△△を使えば事足りるのではないか？
（つらつらと検討したことを記載する）
上記より△△では足りないと判断した。
```

## 結果

```記載後は削除してください。
ADRを適用した後のコンテキストについて記載
良かった点だけでなく悪かった点を含めリストアップすることが望まれる

```

